docs/GettingStartedDocs/Contributors/NonAccMachineGettingStarted.md

```diff
@@ -0,0 +1,228 @@
+# Configure OE SDK on non ACC VM
+
+## Disclaimer
+This document is to provide a viable solution to enable openenclave SGX DCAP remote attestation to run on non Azure Confidential Computing(ACC) machines. It relies on several Intel components and services which are subject to Intel's changes.
+
+## 1. Platform requirements
+- Ubuntu 18.04-LTS 64-bits (should also work on 16.04 LTS but hasn't been tested).
```

Comment:
If this is specific to Linux (not Windows) then please also add Linux to the title on line 1 and filename (or create subdirectories for Linux-only docs / SGX-only docs).

Reply:
I have opened https://github.com/openenclave/openenclave/pull/2979 to try to move around docs and make it more clear which instructions are specific to SGX and which aren't

Reply:
Thanks for the comments from @dthaler and the PR from @jhand2. The filename and the title has been changed accordingly. Files organizing will be done by PR #2979